css/styles.css

```diff
@@ -259,7 +259,7 @@ header {
 header .header-content {
     position: relative;
     width: 100%;
-    padding: 100px 15px;
+    padding: 100px 15px 50px 15px;
```

Comment:
Set padding-top 10px. It's having unnecessary blank space on the top.
```css
padding: 10px 15px 50px 15px;
```

Reply:
Was fix here. f6b8186463e9a6567397b72d003f903adb52f7d1